src/libraries/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs

```diff
@@ -325,6 +325,8 @@ private void InjectHeaders(Activity currentActivity, HttpRequestMessage request)
 
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
             Justification = "The values being passed into Write have the commonly used properties being preserved with DynamicDependency.")]
+        [UnconditionalSuppressMessage("AotAnalysis", "IL3050:AotUnfriendlyApi",
+            Justification = "The values being passed into Write have the commonly used properties being preserved with DynamicDependency.")]
```

Comment:
This is no longer necessary after #76109. It can be reverted.

Reply:
```suggestion
```